File: Programmers/Lv.1/둘만의 암호/155652.js
function solution(s, skip, index) {
    let answer = '';
    
    let alphabets = 'abcdefghijklmnopqrstuvwxyz';
    skip.split('').forEach(v => alphabets = alphabets.replaceAll(v, ''));
    
    s.split('').forEach(v2 => {
        const i = alphabets.indexOf(v2) + index;
        answer += alphabets.charAt(i % alphabets.length);
    });
    
    return answer;
}

File: Programmers/Lv.1/둘만의 암호/155652.java
class Solution {
    public String solution(String s, String skip, int index) {
        String answer = "";
        
        String alphabets = "abcdefghijklmnopqrstuvwxyz";
        for (String delStr : skip.split("")) {
            alphabets = alphabets.replaceAll(delStr, "");
        }
        
        for (String str : s.split("")) {
            int strIndex = alphabets.indexOf(str) + index;
            answer += alphabets.charAt(strIndex % alphabets.length());
        }
        
        return answer;
    }
}
